typescript-selenium-webdriver/README.md

```diff
@@ -1,17 +1,57 @@
+* [Jest](https://jestjs.io/) as our test framework, with [Jest Snapshot Testing](https://jestjs.io/docs/en/snapshot-testing) for baselining
 * [selenium-webdriver](https://www.npmjs.com/package/selenium-webdriver) to automate browsing to the page from the tests
 * [axe-webdriverjs](https://github.com/dequelabs/axe-webdriverjs) to run an axe accessibility scan on the page from the Selenium browser
 * [axe-sarif-converter](https://github.com/microsoft/axe-sarif-converter) and [Sarif.Multitool](https://www.nuget.org/packages/Sarif.Multitool) to compare the scan results to a checked-in baseline
 * [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) to run the tests in a CI build with every Pull Request
 * [Sarif Viewer Build Tab](https://marketplace.visualstudio.com/items?itemName=sariftools.sarif-viewer-build-tab) to visualize the baselined results in Azure Pipelines
+
+## See it in action in Azure Pipelines
+
+[![Build Status](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_apis/build/status/typescript-selenium-webdriver%20CI?branchName=master)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/latest?definitionId=25&branchName=master)
+
+<!--
+  Note to maintainers: The below example images/links come from a specific build instead of the most recent build so we can link to specific tabs.
+  If you update the links such that they point to a different build, make sure to mark that build as Retained so the links don't expire in a month.
+-->
+The accessibility tests run as part of the `yarn test` build step:
+
+[![Screenshot of "yarn test" build logs in sample build](./assets/screenshot-logs-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=1283)
+
+The test pass/fail results display in the Tests tab of the build logs:
+
+[![Screenshot of Tests tab in sample build](./assets/screenshot-tests-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=1283&view=ms.vss-test-web.build-test-results-tab)
+
+Detailed accessibliity scan information also appears in the Scans tab, courtesy of the [Sarif Viewer Build Tab extension](https://marketplace.visualstudio.com/items?itemName=sariftools.sarif-viewer-build-tab):
+
+[![Screenshot of Scans tab in sample build](./assets/screenshot-scans-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=1283&view=sariftools.sarif-viewer-build-tab.sariftools.sarif-viewer-build-tab)
+
+## See it in action on your local machine
+
+1. Install [Chrome](https://www.google.com/chrome/) and [ChromeDriver](http://chromedriver.chromium.org/getting-started), and make sure ChromeDriver is on your PATH.
+1. Clone this sample repository
+
+   ```shell
+   git clone https://github.com/microsoft/axe-pipelines-samples
+   ```
+
+1. Install the dependencies
+
+   ```shell
+   cd ./typescript-selenium-webdriver
+   npm install

```

Comment:
I thought we were using yarn instead of npm in this sample

Reply:
oops, good catch. Will update to clarify that both are fine

Reply:
does `yarn` needs to show up on the Tools and library section? or is that something we consider given or obvious?

Reply:
I think we want to err on the side of stating the obvious for samples like this. `jest` also needs to be globally installed for the local machine instructions to work (or you can `yarn run jest`).